indigo/bulk_creator.py

```diff
@@ -222,6 +222,7 @@ class BaseBulkCreator(LocaleBasedMatcher):
     consolidation_date = None
 
     workflow = None
+    project_topic = None
```

Comment:
The model being added is a taxonomy_topic -- let's be consistent internally with what we call things, even if the user-visible label is a bit different.

Reply:
I used this because we're only using a subset of taxonomy_topics here (the ones that fall under 'projects'); but I'm happy to change it 👍